rust-toolchain

```diff
@@ -1 +1 @@
-nightly
+nightly-2023-03-01
```

Comment:
I'd rather not pin this.

Probably better to remove this file completely.

Reply:
Sorry this was accidental and unrelated to this PR.

Reply:
the file is now gone which should make things easier.